Doc/library/threading.rst

```diff
@@ -137,14 +137,19 @@ This module defines the following functions:
    .. versionadded:: 3.4
 
 
-.. function:: settrace(func)
+.. function:: settrace(func, *, running_threads=False)
 
    .. index:: single: trace function
 
    Set a trace function for all threads started from the :mod:`threading` module.
    The *func* will be passed to  :func:`sys.settrace` for each thread, before its
    :meth:`~Thread.run` method is called.
 
+   If *running_threads* is set, the tracing function will be installed in all running
```

Comment:
```suggestion
   If *running_threads* is true, the tracing function will be installed in all running
```

same remark for setprofile() doc and docstrings.

Wbout "**will be** installed": is the action immediate, or is it "as soon as the thread execution continues"? Anyway, "will be installed" sounds correct to me :-)